### ISSUE TITLE
Safari iOS reloads on initial load if XSLT is not cached

### ISSUE THREAD

**Issue by marksreeves** [BorderTech/wcomponents#126]
This is the infamous "A problem occurred with this webpage so it has been reloaded" issue which can cause all sorts of problems. It went away in iOS 8.4.1 but came back with a vengeance in iOS9.

I have undertaken several experiments with lighter weight JS, no CSS, no style loading and finally discovered that I could reliably solve the issue by manually merging the XSLT.

**Comment by marksreeves** [comment]
The issue is that attempting to load a WComponents application in Safari on iOS 9 causes the load to fail unless all XSLT and JS loaded XML has cached.

@ricksbrown has shown this is more likely caused by the WAI-ARIA rdf import. It still appears to be cache related and the mitigation by merging the XSLT files allows iOS to better handle this import. 

Steps to reproduce:

1. Clear Safari history and data
2. Open a WComponent application
3. **EXPECT**: the application to open <br>
     **ACTUAL**: see a message "A problem occurred with this webpage so it has been reloaded" along with either the loaded application or, more commonly on iOS 9 an indicator that the error has occurred repeatedly,

**NOTE**: This issue occurred in earlier versions of Safari iOS but was not able to be reproduced in iOS 8.4.1 which included several memory management improvements. It has recurred in iOS9.

**Comment by ricksbrown** [comment]
I ended up ruling out the RDF as a cause of this.
I also ruled out JS as the cause.

This problem unfortunately causes the remote debugger to crash so diagnostics on this are not readily available. I even tried remote webkit debugging using Chrome and [ios-webkit-debug-proxy](https://github.com/google/ios-webkit-debug-proxy/) to no avail.

Looks like we will, indeed, have to get rid of any `xsl:include` and `xsl:import` occurrences.
